### ISSUE TITLE
Feature Request - Make identifier/agent searches cap-sensitive

### ISSUE THREAD

**Issue by ewommack** [ArctosDB/arctos#6450]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
When people search for agents/institutions in the search fields now they bring up a ton of names, which is good.
But if all you have is the acronym of an institution or someone's initials it may be really hard to sort through all of the options till you find the correct agent/institution, which is bad.
Example - CAS = California Academy of Sciences, but entering CAS into the agent field brings up anyone with "cas" in their name.

**Describe what you're trying to accomplish** 
Could we have the agent/institution search field be CAP sensitive to help find acronyms and initials that are listed as aka in an Agent profile easier?

From AWG meeting 22 Jun 2023.

**Comment by mkoo** [comment]
Higher Priority needed than wishlist since this is directly impacting users experience with the reorganization of identifiers' CT-thanks!

**Comment by dustymc** [comment]
The current search is case-insensitive CONTAINS.

There is no technical problem with this adjustment, if it's what The Community wants then it can easily be done.

BUT, I very strongly suspect this would have near-absolute negative accessibility impacts. Very few users consider case when searching, very few users will predict how we might have entered some CAS-like name, and very few agents have a CAS-like name. The intersection of those three things is minuscule, and I think this would just mean that almost everybody fails to be helped by the helper. (The helper is very new so people are used to not being helped - I suppose even 'almost never does anything useful' could be considered better than it was, but there's also a significant cost to the queries.)

**Comment by ewommack** [comment]
> The current search is case-insensitive CONTAINS.

So the current search is case sensitive or contains something interesting?

**Comment by mkoo** [comment]
@ewommack no, the current condition is _case-insensitive_ (so treats "CAS" and "cas" the same) and use the SQL _CONTAINS_ which means that any text that has "cas" in it will appear so "Casey" will show up with "CAS" with "Nicasio" (random samples here!)

@dustymc The right answer is that (I think) we can accommodate acronyms in IssuedBy search field already!

I believe if you put the equal size it will give the AKA values you are looking for.
Example, California Academy of Sciences has the AKA of ```CAS```
so in the search box put =CAS and it returns all the specimens with CAS numbers!

So if you want to look up an Issuer by an acronym, just make sure it has an AKA value!
